### DISCUSSION TITLE
Porting the Slint UI Toolkit to a Microcontroller with 264K RAM

### DISCUSSION THREAD

**Discussion by ogoffart** [slint-ui/slint#1117] [Blog Comments]
Discussion thread for the blog post "[Porting the Slint UI Toolkit to a Microcontroller with 264K RAM](https://slint-ui.com/blog/porting-slint-to-microcontrollers.html)"

**Comment by bjorn** [comment]
Well, good thing you renamed "SixtyFPS" before showing off that demo! :wink:

In any case, that is seriously impressive, and it sounds like you'll be able to make it run even better on that small chip eventually!

**Comment by haikyuu** [comment]
Very impressive. I can't wait for support of esp32.
The good thing is that I can start developing on the web in the meantime

**Comment by traverseda** [comment]
Any plans to make this work with micropython? Not sure that's possible, but it seems like a better option than LVGL which does support micropython.

I know there's support for [native machine code in mpy files](https://docs.micropython.org/en/latest/develop/natmod.html). If you got that working it would pretty quickly become the defacto, I think.

**Reply by ogoffart** [reply]
Yes, we do plan to support python (and micropython) in the future.

**Reply by traverseda** [reply]
Awesome, I imagine micropython is quite the challenging target
